docs/guided-onboarding.md

```diff
@@ -1,18 +1,17 @@
 # Collect logs and metrics using the guided onboarding
-The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution for OpenTelemetry Collector.
-To configure the Elastic OTel Collector manually, refer to the [manual configuration](manual-configuration.md) docs.
+The guided onboarding in Kibana or in a serverless Observability project walks you through collecting logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.

```

Comment:
>The guided onboarding in Kibana or in a serverless Observability project

Is this how we differentiate between ESS, Onprem and serverless? Never seen Kibana as the name being exclusive to non-serverless